draft-ietf-httpbis-priority.md

```diff
@@ -813,15 +813,19 @@ might use the relative priority of streams when choosing probe data.
 
 # Fairness {#fairness}
 
-As a general guideline, a server SHOULD NOT use priority information for making
-scheduling decisions across multiple connections, unless it knows that those
-connections originate from the same client. Due to this, priority information
-conveyed over a non-coalesced HTTP connection (e.g., HTTP/1.1) might go unused.
-
-The remainder of this section discusses scenarios where unfairness is
-problematic and presents possible mitigations, or where unfairness is desirable.
+Typically, HTTP implementations depend on the underlying transport to maintain
+fairness between connections competing for bandwidth. When HTTP requests are
+forwarded through intermediaries, progress made by each connection originating
+from end clients can become different over time, depending on how intermediaries
```

Comment:
```suggestion
from user agents can become different over time, depending on how intermediaries
```

Reply:
When talking about intermediaries, I prefer the term end clients